grammars/less.cson

```diff
@@ -273,12 +273,12 @@
     "name": "keyword.other.important.css"
   }
   {
-    'match': '\\$|%|&|\\*|\\/|\\-\\-|\\-|\\+\\+|\\+|~|===|==|=|!=|!==|<=|>=|<<=|>>=|>>>=|<>|<|>|!|&&|\\|\\||\\?\\:|\\*=|(?<!\\()/=|%=|\\+=|\\-=|&=|\\^=|\\/\\b'
+    'match': '\\$|&|\\*|\\/|\\-|\\+|~|=|<=|>=|<|>|&&|\\?\\:|(?<!\\()/=|\\/\\b'
```

Comment:
`&` can't be removed yet because the parent-selector regex doesn't capture standalone `&`s like in the example.
```less
.link {
  & + & {
    color: red;
  }

  & & {
    color: green;
  }

  && {
    color: blue;
  }

  &, &ish {
    color: cyan;
  }
}
```

That's outside the scope of this PR though and can be worked on later.